src/components/CloseBoard/CloseBoard.tsx

```diff
@@ -28,16 +31,30 @@ const CloseBoard = () => {
     if (isCloseBoardPopupActive) popupRef.current?.focus();
   }, [isCloseBoardPopupActive]);
 
+  useEffect(() => {
+    if (top + 200 > window.innerHeight) {
+      dispatch(
+        setCloseBoardPopupPos({
+          top: top - 160,
+          left,
+          boardName,
+          boardId,
+        }),
+      );
+    }
+  }, [isCloseBoardPopupActive]);
```

Comment:
You are using isCloseBoardPopupActive as a depandancy but do not use in useEffect callback function